src/nvim/options.lua

```diff
@@ -2567,7 +2567,7 @@ return {
       vi_def=true,
       expand=true,
       varname='p_vdir',
-      defaults={if_true={vi=macros('DFLT_VDIR')}}
+      defaults={if_true={vi=''}}
```

Comment:
Yup, I plan to do all of it. Just want to make sure that everyone else agrees with this approach first.